line_following:
  ros__parameters:
    desired_value: 35.0
    Kp: 0.01
    Ki: 0.00
    Kd: 0.00
    linear_x_min: 0.0
    linear_x_max: 1.0
    angular_z_min: -0.5
    angular_z_max: 0.5

